src/shared/components/proteinChainPanel/ProteinChainPanel.tsx

```diff
@@ -206,6 +206,10 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
         );
     }
 
+    componentDidMount() {
+        onNextRenderFrame(() => this.props.store.pdbChainDataStore.selectFirstChain());
```

Comment:
Probably there is a better solution to select the first chain at init, but for now keeping the existing logic.

Reply:
what's reason to use onNextRenderFrame?

Reply:
i can see it's been this way, but just curious.

Reply:
For some reason, it doesn't re-render when we call `selectFirstChain` without `onNextRenderFrame`. Ideally we shouldn't be calling `selectFirstChain` at init at all, but I think that should be fixed in another PR.